modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+[id="nw-sriov-cfg-bond-interface-with-virtual-functions_{context}"]
+= Configuring a bond interface from two SR-IOV interfaces
+
+Bonding enables multiple network interfaces to be aggregated into a single logical "bonded" interface. Bond-CNI brings bond capability into containers.
+
+Bond-CNI can be created using SR-IOV virtual functions and placing them in the container network namespace.
+
+Red Hat OpenShift Platform only supports Bond-CNI using SR-IOV virtual functions. Physical interfaces are not supported.
+
+== Pre-requisites
+
+The SR-IOV operator must be installed  and configured in order to obtain virtual functions in a container.
+
+To configure SR-IOV interfaces, an SR-IOV network and policy must be created for each interface.
+
+The SR-IOV operator creates a network attachment definition for each SR-IOV interface, based on the SR-IOV network and policy defined.
+
+== Creating a bond network attachment definition
+
+Now that the SR-IOV virtual functions are available, you can create a bond network attachment definition.
+
+[source,yaml]
+----
+    apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: bond-net1
+      namespace: demo
+    spec:
+      config: '{
+      "type": "bond", <1>
+      "cniVersion": "0.3.1",
+      "name": "bond-net1",
+      "ifname: "bond0" <2>
+      "mode": "active-backup",
+      "failOverMac": 1,
```

Comment:
"failOverMac": 1, is must for active-backup mode. since we support only active-backup we shoul mension that:
 https://github.com/k8snetworkplumbingwg/bond-cni/blob/master/README.md?plain=1#L35